src/app/shared/service/tableid.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
```

Comment:
File name "tableid.service.ts" is not understandable. Please give a proper file naming